openshift/keycloak.app.yaml

```diff
@@ -106,6 +106,11 @@ objects:
           - name: JAVA_OPTS
             value: >-
               -server -Xms256m -Xmx4096m -XX:MetaspaceSize=96M
+              -XX:+UseParallelGC
+              -XX:MinHeapFreeRatio=20
+              -XX:MaxHeapFreeRatio=40
+              -XX:GCTimeRatio=4
```

Comment:
OK. I found the source of that recommended settings - https://developers.redhat.com/blog/2014/07/22/dude-wheres-my-paas-memory-tuning-javas-footprint-in-openshift-part-2/
Sounds reasonable :)